### ISSUE TITLE
[Question]: how to change the color for modified lines ?

### ISSUE THREAD

**Issue by UffeJakobsen** [eranif/codelite#3439]
### What happened?

[Question]: how to change the line number margin color for modified lines ?

### Version

Self compiled

### Operating system

Linux

**Comment by eranif** [comment]
I don't think you can... can you post a screenshot so I can confirm that you are asking about what I think you are asking? :)

**Comment by UffeJakobsen** [comment]
The circled color on line #19 is the result of of saving changes on that line - this is the color I would like to change

(line #23 is current line)

![bbb](https://github.com/user-attachments/assets/3f0cacc9-6815-41c8-84b2-218e92919a9b)
